specification/azurestack/resource-manager/Microsoft.AzureStack/preview/2020-06-01-preview/EdgeSubscription.json

```diff
@@ -0,0 +1,403 @@
+            },
+            {
+              "$ref": "#/parameters/ApiVersionParameter"
+            }
+          ],
+          "responses": {
+            "200": {
+              "description": "OK",
+              "schema": {
+                "$ref": "#/definitions/EdgeSubscription"
+              }
+            },
+            "default": {
+              "description": "Error response describing why the operation failed.",
+              "schema": {
+                "$ref": "#/definitions/ErrorResponse"
+              }
+            }
+          }
+        },
+        "delete": {
+          "x-ms-examples": {
+            "Delete the requested Azure Stack edge subscription.": {
+              "$ref": "examples/EdgeSubscription/DeleteEdgeSub.json"
+            }
+          },
+          "tags": [
+            "EdgeSubscriptions"
+          ],
+          "description": "Delete the requested Azure Stack edge subscription.",
+          "operationId": "EdgeSubscriptions_Delete",
+          "parameters": [
+            {
+              "$ref": "#/parameters/SubscriptionIdParameter"
+            },
+            {
+              "$ref": "#/parameters/ResourceGroupNameParameter"
+            },
+            {
+              "$ref": "#/parameters/EdgeSubscriptionNameParameter"
+            },
+            {
+              "$ref": "#/parameters/ApiVersionParameter"
+            }
+          ],
+          "responses": {
+            "200": {
+              "description": "OK"
+            },
+            "204": {
+              "description": "NO CONTENT"
+            },
+            "default": {
+              "description": "Error response describing why the operation failed.",
+              "schema": {
+                "$ref": "#/definitions/ErrorResponse"
+              }
+            }
+          }
+        },
+        "put": {
+          "x-ms-examples": {
+            "Create or update an Azure Stack registration.": {
+              "$ref": "examples/EdgeSubscription/PutEdgeSub.json"
+            }
+          },
+          "tags": [
+            "EdgeSubscriptions"
+          ],
+          "description": "Create a new edge subscription",
+          "operationId": "EdgeSubscriptions_Create",
+          "parameters": [
+            {
+              "$ref": "#/parameters/SubscriptionIdParameter"
+            },
+            {
+              "$ref": "#/parameters/ResourceGroupNameParameter"
+            },
+            {
+              "$ref": "#/parameters/EdgeSubscriptionNameParameter"
+            },
+            {
+              "$ref": "#/parameters/ApiVersionParameter"
+            },
+            {
+              "$ref": "#/parameters/NewEdgeSubscriptionParameter"
+            }
+          ],
+          "responses": {
+            "200": {
+              "description": "OK",
+              "schema": {
+                "$ref": "#/definitions/EdgeSubscription"
+              }
+            },
```

Comment:
same as above - need an operation-specific description for status codes, this applies throughout

Reply:
Updated.